filament/include/filament/Texture.h

```diff
@@ -236,6 +236,14 @@ class UTILS_PUBLIC Texture : public FilamentAPI {
          */
         Builder& import(intptr_t id) noexcept;
 
+        /**
+         * Specifies the sample count of the texture
+         *
+         * @param sampleCount sample count
+         * @return This Builder, for chaining calls.
+         */
+        Builder& samples(uint8_t sampleCount) noexcept;
```

Comment:
How are the values of 0 and 1 should be interpreted by a client of the lib? The backend distinguishes 0 (implicitly resolved) and 1 (request not to multisample). Is 0 sanitized to 1 somewhere down the line?

Reply:
Fixed in https://github.com/shapr3d/filament/pull/67/commits/917a91248e3bd5fdf063c55e3dcdbf473fc76d29: An assert checks now the zero case. The automatic sample count have only meaning in case of the frame graph